packages/cron/src/jobs/pins.js

```diff
@@ -0,0 +1,105 @@
+import debug from 'debug'
+import { gql } from '@web3-storage/db'
+import { toPinStatusEnum } from '../../../api/src/utils/pin.js'
```

Comment:
We should migrate the package names to the `@web3-storage` namespace to avoid any potential for suprise squatting. npm workspace _should_ resolve internally first, but, you know, bugs happen.

Reply:
https://github.com/web3-storage/web3.storage/issues/86